i18n/ja.json

```diff
@@ -48,10 +48,6 @@
     "translation": "9月"
   },
   {
-    "id": "api.admin.add_certificate.array.app_error",
```

Comment:
yes @kaakaa that's ok cause it was removed from `en.json` by mistake I think, I'll add it back for a future PR